contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -0,0 +1,155 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+from pants.base.build_environment import get_buildroot
+from pants.base.exceptions import TaskError
+from pants.base.workunit import WorkUnitLabel
+from pants.util.contextutil import pushd
+from pants.util.memo import memoized_method
+
+from pants.contrib.node.targets.node_package import NodePackage
+from pants.contrib.node.tasks.node_task import NodeTask
+
+
+class JavascriptStyle(NodeTask):
+  """ Check javascript source files to ensure they follow the style guidelines.
+
+  :API: public
+  """
+
+  _JS_SOURCE_EXTENSION = '.js'
+  _JSX_SOURCE_EXTENSION = '.jsx'
+  INSTALL_JAVASCRIPTSTYLE_TARGET_NAME = 'synthetic-install-javascriptstyle-module'
+
+  def __init__(self, *args, **kwargs):
+    super(JavascriptStyle, self).__init__(*args, **kwargs)
+
+  @classmethod
+  def register_options(cls, register):
```

Comment:
Hey: sorry, late-breaking requirement. Please add a `--transitive` flag with an implementation identical to this one:
https://github.com/pantsbuild/pants/blob/65fb5f420c9a949cbc781861b240258c30639eac/src/python/pants/backend/jvm/tasks/scala_rewrite_base.py#L51

For performance reasons during `arc lint`, we're going to need to run intransitively.